navbar.html

```diff
@@ -42,7 +42,11 @@
         </div>
         <div br-slot="br-navbar-toolbar">
           <div class="navbar-right">
-            <div br-slot="br-navbar-tool"></div>
+            <div class="btn-toolbar">
```

Comment:
Does this force tools to be buttons? Perhaps we want the toolbar to have  a slot for tools and a built-in button toolbar with a different slot for each toolbar-button?